bionic/flow.py

```diff
@@ -1817,12 +1819,62 @@ def core__persistent_cache(
 
     @builder
     @decorators.immediate
-    def core__executor(
+    def core__process_executor(
         core__parallel_execution__enabled,
         core__parallel_execution__worker_count,
     ):
         if not core__parallel_execution__enabled:
             return None
-        return Executor(core__parallel_execution__worker_count)
+        return ProcessExecutor(core__parallel_execution__worker_count)
+
+    builder.assign("core__aip_execution__enabled", False, persist=False)
+    builder.assign("core__aip_execution__gcp_project", None, persist=False)
+    builder.assign("core__aip_execution__docker_image", "bionic:latest", persist=False)
+
+    @builder
+    @decorators.immediate
+    def core__aip_execution__job(
+        core__aip_execution__enabled,
+        core__aip_execution__gcp_project,
+        core__aip_execution__docker_image,
+    ):
+        if not core__aip_execution__enabled:
+            return None
+        if core__aip_execution__gcp_project is None:
+            raise AssertionError(
+                "core__aip_execution__gcp_project is None, " "but needs a value"
```

Comment:
Should this be one single string?

Also, why can we explain why it needs a value?

Reply:
I modeled this after `core__persistent_cache__cloud_store` to be consistent, but I can make it a single string. I'll add an explanation, but let's wait till we resolve the [decorator question](https://github.com/square/bionic/pull/253#discussion_r487072298). Depending on its result, we might not even create this here.

Reply:
I'm guessing the string for `core__persistent_cache__cloud_store` used to be on separate lines, but they got moved by `black`. AFAICT there's no reason for it to use separate strings.